public/sitemap.xml

```diff
@@ -1,289 +1,9 @@
-﻿<?xml version="1.0" encoding="UTF-8"?>
-		<!--	created with www.mysitemapgenerator.com	-->
-		<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+﻿<?xml version="1.0" encoding="UTF-8" ?>
+<!--	created with www.mysitemapgenerator.com	-->
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
 <url>
 	<loc>https://app.socious.io/</loc>
```

Comment:
actually this route is not valid please put the app.socious.io/jobs

Reply:
No I removed 'app.socious.io/jobs' from sitemap on purpose. We do not need to index this page. what we need to be indexed is job details page which I call indexing api for them. When we index 'app.socious.io/jobs' and more URLs, google consider some of our URLs repetitive and refuse to index them